### PR TITLE
iTerm 3 nightly compatibility closes #89

### LINKED ISSUE #89: iTerm nightly compatibility
I run iTerm nightly (using `homebrew cask`) but I'd been having issues with Sublime Terminal. I [reported it the iTerm developer George Nachman](https://gitlab.com/gnachman/iterm2/issues/3718#note_1724190) and he cleaned up the iTerm.sh file. His version opens in a new tab of the current terminal by default, which I consider to be the best anyway.

The cleaned up version is available at https://gist.github.com/gnachman/b9ae9cf06c44b5a95edf and it seems reasonable to use it, but I'll hold off for feedback before doing a pull request.

### PR DESCRIPTION
**Pull request by jcrben**
You'll have to hold off on merging this as it apparently is not working with iTerm 2, but at least we can easily see the differences. I hope to look into this a bit more tomorrow.

### CODE REVIEW

**Comment by twolfson** [comment]
Alright, cool. Let us know when both versions are compatible. If feature detection isn't possible, then I think we will be fine with version detection (although not as favorable).

File: iTerm.sh [hunk 1]
```diff
@@ -32,41 +32,40 @@ fi
 if (( ! $RUNNING )); then
 	osascript<<END
 	tell application "iTerm"
-		tell current terminal
-			launch session "Default Session"
-			tell the last session
-				write text "$CD_CMD"
-			end tell
-		end tell
+            tell current window
```
Comment (twolfson) [inline iTerm.sh]: One thing to note is the indentation for this file was tabs. Can we keep the new lines of codes indented with tabs as well for consistency?

**Comment by pzgz** [comment]
How about a separated bash file to launch iterm2 v3 version, and user can use user settings to use the right bash file for launching, I will open a new PR for that, as an option.

**Comment by twolfson** [comment]
I believe @wbond might be building something to repair this at a lower level so I will wait for him to reply:

https://github.com/wbond/sublime_terminal/issues/94

**Comment by mbigras** [comment]
Hello,
Currently running:
iTerm2: Build 2.9.20160523 with shell integration and ST3 build 3114. Currently the sublime_terminal package isn't opening iTerm2 for me after modifying the settings file by adding "iTerm.sh". Also I replaced my iTerm.sh with the one that @twolfson suggested but it still doesn't work. Leaving it blank still opens with the default Terminal program as expected. I see #94 is suggesting to rebuild using a [new function launch_terminal](https://github.com/codexns/newterm). Has `launch_terminal` been implemented yet? Looking through the logs and through Terminal.py it seems like no. Is there currently a way to open iTerm2 from sublime using this package or any others? Thanks :)

**Comment by twolfson** [comment]
To remove potential issues between iTerm and iTerm2, we have gone with separate scripts via #98. Thanks for your work on this and sorry for the huge delay -- I thought we would have been on another repo by now but I guess not =/

**Pull request closed without merging**